.github/workflows/scorecards-analysis.yml

```diff
@@ -20,15 +20,15 @@ jobs:
       security-events: write
       actions: read
       contents: read
-
+      id-token: write
```

Comment:
Why does action need this permission?

Reply:
We use GitHub's OIDC token to sign the results which ensures integrity of the result displayed in the badge.

Reply:
Would you mind adding a comment to that effect? I realize the other permissions aren't documented, but it'd be nice to (eventually) have them all documented so that future maintainers understand the security scope of the workflow.

Reply:
Done.

Reply:
Thanks! I'll admit that I'm still a little murky on what this means -- does it mean that the Scorecard server is verifying the OIDC-minted JWT to confirm that the results are actually being created from a particular run of this workflow?

Reply:
Yep, that's correct.

Reply:
Thanks for clarifying!